app/controllers/application_controller.rb

```diff
@@ -6,4 +6,8 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
+
+  def require_user
```

Comment:
Killin two birds with one stone; we would've needed this eventually, thanks for doing that